### PR TITLE
Add electronic signature consent wording

### PR DESCRIPTION
**Pull request by squirrelo**
This adds the "I consent to electronic signature" button requested by IRB.

### CODE REVIEW

File: amgut/lib/locale_data/american_gut.py [hunk 1]
```diff
@@ -273,7 +273,8 @@
     'ADD_NEW': 'Add a New Human Sample Source',
     'EXPLANATION': 'You have entered the add human source workflow. During this workflow you will add a human source that represents whoever is being sampled. You be asked for consent to join the project and then asked survey questions.',
     'ONCE_ADDED': 'Once you have added a human source, you will then see the name of that source in the left menu, and you will also have an option for adding a sample to that source. When you click that, you will be able to select the appropriate barcode and add sample metadata.',
-    'CONTINUE': 'Continue'
+    'ELECTRONIC_SIGNATURE': 'In order to participate in this study, you will need to sign a research consent form. This can be done electronically. To consent to using an electronic signature, please click the button below.',
+    'ELECTRIC_SIG_CONSENT': 'I consent to using an electronic signature'
```
Comment (adamrp) [inline amgut/lib/locale_data/american_gut.py]: `ELECTRIC`? Sounds funny.

File: amgut/lib/locale_data/american_gut.py [hunk 2]
```diff
@@ -273,7 +273,8 @@
     'ADD_NEW': 'Add a New Human Sample Source',
     'EXPLANATION': 'You have entered the add human source workflow. During this workflow you will add a human source that represents whoever is being sampled. You be asked for consent to join the project and then asked survey questions.',
     'ONCE_ADDED': 'Once you have added a human source, you will then see the name of that source in the left menu, and you will also have an option for adding a sample to that source. When you click that, you will be able to select the appropriate barcode and add sample metadata.',
-    'CONTINUE': 'Continue'
+    'ELECTRONIC_SIGNATURE': 'In order to participate in this study, you will need to sign a research consent form. This can be done electronically. To consent to using an electronic signature, please click the button below.',
```
Comment (adamrp) [inline amgut/lib/locale_data/american_gut.py]: Should we wait on official wording from Machelle?

File: amgut/lib/locale_data/british_gut.py [hunk 3]
```diff
@@ -465,7 +465,9 @@
     'ADD_NEW': "Add a New Human Sample Source",
     'CONTINUE': "Continue",
     'EXPLANATION': "You have entered the add human source workflow. During this workflow you will add a human source that represents whoever is being sampled. You be asked for consent to join the project and then asked survey questions.",
-    'ONCE_ADDED': "Once you have added a human source, you will then see the name of that source in the left menu, and you will also have an option for adding a sample to that source. When you click that, you will be able to select the appropriate barcode and add sample metadata."
+    'ONCE_ADDED': "Once you have added a human source, you will then see the name of that source in the left menu, and you will also have an option for adding a sample to that source. When you click that, you will be able to select the appropriate barcode and add sample metadata.",
+    'ELECTRONIC_SIGNATURE': 'In order to participate in this study, you will need to sign a research consent form. This can be done electronically. To consent to using an electronic signature, please click the button below.',
+    'ELECTRIC_SIG_CONSENT': 'I consent to using an electronic signature'
```
Comment (adamrp) [inline amgut/lib/locale_data/british_gut.py]: We probably don't want to delay, but if it's quick, is it worth running this new text by a "native british speaker"?

File: amgut/lib/locale_data/american_gut.py (hunk 2 above)
Comment (squirrelo) [inline amgut/lib/locale_data/american_gut.py]: Yeah, I think we can. I'll update the pull request once I have it.

File: amgut/lib/locale_data/american_gut.py (hunk 1 above)
Comment (squirrelo) [inline amgut/lib/locale_data/american_gut.py]: fixed

**Comment by adamrp** [comment]
I think this will be fine, just wondering about wording. Also, Machelle said

>The subject must agree to use the electronic format such as by clicking an “I agree” icon

I'm assuming that the button that says "I consent to using an electronic signature" will suffice here, and we don't *actually* need an icon/image.

**Comment by adamrp** [comment]
:+1: once we get the official wording, which will include:
* A clear description of the subject’s rights
* A description of the procedure to withdraw consent for the electronic record

**Pull request merged**